js/runner.go

```diff
@@ -194,8 +197,9 @@ func (r *Runner) newVU(samplesOut chan<- stats.SampleContainer) (*VU, error) {
 		Console:        r.console,
 		BPool:          bpool.NewBufferPool(100),
 		Samples:        samplesOut,
-		m:              &sync.Mutex{},
+		RunMutex:       &sync.Mutex{},
 	}
+	vu.Runtime.Set("__VU", vu.ID)
```

Comment:
This closes https://github.com/loadimpact/k6/issues/889 ?

Reply:
No, `__VU` is still undefined in init. This just moved it from `Reconfigure()`, which no longer exists.

Reply:
Hmm this should be easy enough to fix now, I think. Now that we pass the VU ID to `Runner.newVU()`, we can just pass it to `Bundle.Instantiate()` as well, and set the `__VU` value there. We can set it in the newly created goja runtime there, so it would be available before the init context is executed. Anyway, this should be fixed in a separate PR, so I'll make a note in the issue and resolve this.